doc/user/user/analysis-visualizations/index.md

```diff
@@ -11,42 +11,84 @@ IRIDA Analysis Visualizations
 
 IRIDA has a set of integrated visualization tools that allow for a more thorough analyses of pipeline results.
 
-### Phylogenetic Tree Visualization
+## Phylogenetic Tree Visualization
 
-Phylogenetic trees created by the SNVPhyl Analysis Pipeline are combined with metadata from the samples in the analysis to display the metadata beside the tree leaves.
+Phylogenetic trees produced by analysis pipelines, for example SNVPhyl, are displayed on the analysis page in the "Phylogentic Tree" Tab. Sample metadata can be displayed concurrently with the tree and can be toggled on or off (either on a field or metadata template basis).
 
-![Tree](images/plain_tree.png)
+![Tree](images/tree-viewer.png)
 
-Sample metadata can be added to the sample using the sample metadata uploader (see [Sample Metadata](../sample-metadata) for how to import).
+If you want to display additional metadata, first upload it using the sample metadata uploader (see [Sample Metadata](../sample-metadata) for how to import) or by clicking the sample name in the "Settings" -> "Samples" tab and going to the "Metadata" tab.
 
-To get to the advanced visualization page, on the analysis page, click the `View Advanced Visualization` button on the `Phylogenetic Tree` tab:
+### Tree Menu Option (at top right of tree area)
+#### Toggling Tree Types
 
-![Link to advanced analysis on analysis results page](images/viz_link.png)
+The phylogenetic tree viewer can display five types of trees:
 
-#### Using Sample Metadata Field Templates
+1. Circular
+1. Diagonal
+1. Hierarchical
+1. Radial
+1. Rectangular (default)
 
-[Sample Metadata Templates](../sample-metadata-templates/) can be used to update which metadata fields are displayed in the visualization.  When the page is loaded the default template is presented, all metadata fields are shown.  A different template can be selected from the template selection as demonstrated below.  This will update the view with the desired metadata fields in the proper order.
+![Tree Shape Selection](images/tree-shape-selection.gif)
 
-![Demonstration of applying Sample Metadata Templates to the visualization](images/template_selection.png)
+You can change the tree shape by clicking on the tree shape menu icon at the top right corner of the tree area.  A drop down menu will appear allowing you to select the desired tree shape.
```

Comment:
Can we add an image here of what the icon looks like? It can just me an inline image, Just incase someone doesn't watch the gif atleast it will be clear to them of what they need to click on